src/vaadin-date-picker-mixin.html

```diff
@@ -341,7 +346,10 @@
       this.addEventListener('keydown', this._onKeydown.bind(this));
       this.addEventListener('input', this._onUserInput.bind(this));
       this.addEventListener('focus', e => this._noInput && e.target.blur());
-      this.addEventListener('blur', e => !this.opened && this.validate());
+      this.addEventListener('blur', e => {
+        this.autoOpenDisabled && this._selectAndValidateInput();
+        return !this.opened && this.validate();
```

Comment:
Using `return` does nothing even if we return `false`, because [blur event](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event) is not cancelable. 
So please remove `return` and keep the line as it was.

Reply:
Done